src/main/groovy/com/google/protobuf/gradle/ProtobufExtract.groovy

```diff
@@ -43,21 +49,29 @@ class ProtobufExtract extends DefaultTask {
    */
   private File destDir
   private Boolean isTest = null
+  private final ConfigurableFileCollection inputFiles = project.files()
 
   public void setIsTest(boolean isTest) {
     this.isTest = isTest
   }
 
+  @Input
   public boolean getIsTest() {
     Preconditions.checkNotNull(isTest)
     return isTest
   }
 
+  @InputFiles
+  @PathSensitive(PathSensitivity.NAME_ONLY)
```

Comment:
I am not sure about the `NAME_ONLY`. Are you should or is `RELATIVE` safer?

It seems like the task is copying/extracing from the input files. Maybe it should have a copy-spec instead of plain input files instead?

It also seems like archives can be part of the inputFiles.

For now, using `NAME_ONLY` does not change the behavior, since that is what the plugin declared before.

Reply:
Exactly. Let's not change behavior where we don't have to for now.

Reply:
Please add a TODO here to revisit the correctness of PathSensitivity.